primitives/runtime/src/bounded/bounded_vec.rs

```diff
@@ -290,9 +290,12 @@ impl<T: Decode, S: Get<u32>> Decode for BoundedVec<T, S> {
 	}
 }
 
-// `BoundedVec`s encode to something which will always decode as a `Vec`.
+// `BoundedVec` encode to something which will always decode as a `Vec`.
 impl<T: Encode + Decode, S: Get<u32>> EncodeLike<Vec<T>> for BoundedVec<T, S> {}
 
+// `BoundedVec` encodes to something which will always decode as a `Slice`.
```

Comment:
Changed the comment cause you can't decode to slice as it isn't an owned value. As I understand all that matters is that they encode to the same thing